src/components/BackgroundImage/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+interface BackgroundImageProps {
+  children: React.ReactNode;
+  backgroundUrl?: string;
+  'data-test-id'?: string;
+}
+
+const BackgroundImage = ({ backgroundUrl, children, ...attributes }: BackgroundImageProps): JSX.Element => {
+  return backgroundUrl ? (
+    <div className={`h-screen w-screen relative`} {...attributes}>
```

Comment:
You can use this [`classnames`](https://www.npmjs.com/package/classnames) package to merge classes like you did in `TextInput` component